tests/integration/security/authorization_test.go

```diff
@@ -1061,61 +1062,64 @@ func TestAuthorization_GRPC(t *testing.T) {
 	framework.NewTest(t).
 		Features("security.authorization.grpc-protocol").
 		Run(func(t framework.TestContext) {
-			ns := namespace.NewOrFail(t, t, namespace.Config{
-				Prefix: "v1beta1-grpc",
-				Inject: true,
-			})
-			var a, b, c, d echo.Instance
-			echoboot.NewBuilder(t).
-				With(&a, util.EchoConfig("a", ns, false, nil)).
-				With(&b, util.EchoConfig("b", ns, false, nil)).
-				With(&c, util.EchoConfig("c", ns, false, nil)).
-				With(&d, util.EchoConfig("d", ns, false, nil)).
-				BuildOrFail(t)
+			ns := apps.Namespace1
+			c := apps.C.Match(echo.Namespace(apps.Namespace1.Name()))
+			d := apps.D.Match(echo.Namespace(apps.Namespace1.Name()))
+			for _, a := range []echo.Instances{apps.A.Match(echo.Namespace(apps.Namespace1.Name())), apps.VM.Match(echo.Namespace(apps.Namespace1.Name()))} {
```

Comment:
can you put `[]echo.Instances{apps.A.Match(echo.Namespace(apps.Namespace1.Name())), apps.VM.Match(echo.Namespace(apps.Namespace1.Name()))}` in a separate variable out of the loop? Make it a bit easier to read, same below.

Reply:
Done.